framework/src/st_things/things_stack/framework/things_sss_manager.c

```diff
@@ -306,5 +306,15 @@ bool things_encrypt_artik_uuid(unsigned char *output)
 	memcpy(output, encode_buf2, 8);
 	THINGS_LOG_D(THINGS_DEBUG, TAG, "output len [%d][%s]", strlen((char *)output), output);
 
+	// 6. url safe
+	int i;
+	for (i = 0; i < 8; i++) {
+		if (output[i] == '+')
```

Comment:
Please use braces even if it has single statement

Reply:
Ok. Thanks.
i will fix and commit again.